ingestion/us/chicago.py

```diff
@@ -0,0 +1,93 @@
+from dateutil.parser import parse as parse_date
+from ingestion.data_models import UpstreamDataset
+from time import sleep
+import httpx
+import json
+
+"""
+Ingestion script for Socrata-based Chicago data portal.
+"""
+
+# URL below pulls up 100 dataset records at preferred offset
+CATALOG = "https://data.cityofchicago.org/api/catalog/v1?explicitly_hidden=false&limit=100&offset={}&order=page_views_total&published=true&q=&search_context=data.cityofchicago.org&show_unsupported_data_federated_assets=false&tags=&approval_status=approved&audience=public"
+
+
+# to be pulled from new utils
+def make_request(url):
+    """ping the catalog url"""
+    ping = httpx.get(url)
+    ping.raise_for_status()
+
+    return ping
+
+
+def extract_updata(catalog):
```

Comment:
Important: this file seems to not match the function interfaces expected in `ingest.py`